rust-code-analysis-book/src/metrics.md

```diff
@@ -1,24 +1,29 @@
 # Supported Metrics
 
-**rust-code-analysis** implements a series of metrics
-- **CC**: it calculates the code complexity examining the
+**rust-code-analysis** implements a series of metrics:
+
+- **ABC**: it measures the size of a source code by counting the number of
+Assignments (`A`), Branches (`B`) and Conditions (`C`).
+- **BLANK**: it counts the number of blank lines in a source file.
+- **CC**: it calculates _Cyclomatic_ complexity examining the
   control flow of a program.
-- **SLOC**: it counts the number of lines in a source file.
-- **PLOC**: it counts the number of physical lines (instructions) contained in
-  a source file.
-- **LLOC**: it counts the number of logical lines (statements) contained in
-  a source file.
 - **CLOC**: it counts the number of comments in a source file.
-- **BLANK**: it counts the number of blank lines in a source file.
+- **COGNITIVE**: it calculates _Cognitive_ complexity examining whether a code
```

Comment:
I'd add a link to the definition of cognitive complexity, and change the description to be:
> it calculates the _Cognitive_ complexity, measuring how complex it is to understand a unit of code.